SingularityUI/app/components/tasks/Columns.jsx

```diff
@@ -0,0 +1,302 @@
+import React, { PropTypes } from 'react';
+import Column from '../common/table/Column';
+
+import Utils from '../../utils';
+
+import JSONButton from '../common/JSONButton';
+import Glyphicon from '../common/atomicDisplayItems/Glyphicon';
+
+export const TaskId = (
+  <Column
+    label="Task ID"
+    id="taskId"
+    key="taskId"
```

Comment:
Mmmm I could see it either way. I'm not sure if using key for other purposes is an antipattern or not. But if not it might be useful. Your call on that.